java/bundles/org.eclipse.set.feature.table/src/org/eclipse/set/feature/table/internal/TableServiceContextFunction.java

```diff
@@ -87,7 +89,7 @@ public Object compute(final IEclipseContext context,
 		tableService = ContextInjectionFactory.make(TableServiceImpl.class,
 				context);
 		modelServiceMap.keySet().forEach(
-				elementId -> tableService.addModelServiceById(elementId,
+				elementId -> tableService.addModelServiceByInfo(elementId,
 						modelServiceMap.get(elementId)));
```

Comment:
@TruongQuangSB since there is not an elementId the key inside the model service map I would recomment to rename the variable here to

```suggestion
				tableInfo -> tableService.addModelServiceByInfo(tableInfo,
						modelServiceMap.get(tableInfo)));
```